tensorrt_llm/llmapi/llm_args.py

```diff
@@ -326,6 +326,7 @@ class EagleDecodingConfig(DecodingBaseConfig):
     num_eagle_layers: Optional[int] = None
     max_non_leaves_per_layer: Optional[int] = None
     eagle3_one_model: Optional[bool] = True
+    num_capture_layers: Optional[int] = 3
```

Comment:
Type annotation should be `int` if it can't be `None`.

Why are we making this part of the part of the spec config? EAGLE3 should capture hidden states from 3 layers. It's not eagle 3 if it doesn't do that